plugins/catalog-backend-module-ldap/src/ldap/config.ts

```diff
@@ -426,6 +462,7 @@ export function readProviderConfigs(config: Config): LdapProviderConfig[] {
         return mergeWith({}, defaultGroupConfig, it, replaceArraysIfPresent);
       }),
       schedule,
+      vendor: readVendorConfig(c.getOptionalConfig('vendor')),
```

Comment:
Any config change needs to also have the associated `config.d.ts` file schema update in the root of this package. The file is a bit messy but in particular [inside this block](https://github.com/backstage/backstage/blob/c40aa8798c437fd8b89ac84da0a4b916bd5d4bf2/plugins/catalog-backend-module-ldap/config.d.ts#L435) you need to add the new structure.